add more details about realm definition (#191)
decentraland/adr: content/ADR-128-realm-modifiers-for-worlds-in-renderer.md

Discoverability of content will be outside-in, as content creators will be able to share easy to remember URLs to enter the World directly, or inside-out using a join command. Anybody will be able to join a World.

## Solution Space Exploration

> To be able to modify the features necessary in each world, we are developing a Realms Plugin which contains Realms Modifiers. Each of these modifiers will have the responsibility to alter DCL functionality by checking if we are entering a developer-generated realm or a DAO realm.

Reviewer: ```suggestion
To modify the features necessary in each world, we are developing a Realms Plugin which contains Realms Modifiers. Each will be responsible for altering DCL functionality by checking if we are entering a developer-generated realm or a DAO one.
```

Controls the functionality associated to the minimap when entering a Realm. It determines if the minimap should be visible using the `realmConfiguration.Configurations.Minimap.Enabled` value. Also, it determines if the Jump To Home Button should be present. This button will appear on any given user world, and it will take the user back to the home location in the most populated DAO Catalyst.
When needed, this modifier will implement the change of the minimap image; as a user generated world will have the possibility to show its own minimap png.

### Data Structure

> `AboutResponse` contains the info of the realm which has just been accessed. This class is autogenerated from protobuff specifications and adheres to [ADR-110: Realm description](https://adr.decentraland.org/adr/ADR-110). Each modifier gets access to the `AboutResponse`, and gets the info necessary to execute its behavior. It has the following structure:

Reviewer: ```suggestion
`AboutResponse` contains the info of the realm which has just been accessed. This class is autogenerated from protobuf specifications and adheres to [ADR-110: Realm description](https://adr.decentraland.org/adr/ADR-110). Each modifier gets access to the `AboutResponse` and gets the info necessary to execute its behavior. It has the following structure:
```
